src/category_theory/conj.lean

```diff
@@ -111,10 +111,10 @@ by simp only [conj_Aut_apply, iso.trans_symm, iso.trans_assoc]
 conj_Aut_mul α g f
 
 @[simp] lemma conj_Aut_pow (f : Aut X) (n : ℕ) : α.conj_Aut (f^n) = (α.conj_Aut f)^n :=
-is_monoid_hom.map_pow α.conj_Aut f n
+α.conj_Aut.to_monoid_hom.map_pow f n
```

Comment:
Not sure if we should also introduce shortcuts for all the equiv types... Let's not do that in this PR.